portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,6 +43,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = getParameter(request, "comment", "");
     comments.add(comment);
+
+    long timestamp = System.currentTimeMillis();
+
+    Entity commentEntity = new Entity("Comment");
```

Comment:
Can we create a data class for `Comment`? And then we can delegate the datastore logic to the `Comment` class. E.g.
```
Comment comment = new Comment(...)
datastore.put(comment.toEntity())
```
Right now the servlet seems to be having too much responsibility

Reply:
Indeed! I will add it later.😅

Honestly, thought these pull requests are changes made based on the tutorial. I wanted to make sure that the bare minimum works before I make further changes. I thought that this is the way to do things. Would you prefer me to abstract things along the way rather than to leave it till after I complete the tutorials?

Reply:
Totally agree that we should have MVP first, please leave a TODO here if you prefer to do that in a later stage.

Reply:
Okay! I'll add a todo. (: